src/Tizen.NUI/src/internal/Interop/Interop.PixelData.cs

```diff
@@ -34,6 +34,9 @@ internal static partial class PixelData
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_PixelData_GetPixelFormat")]
             public static extern int PixelData_GetPixelFormat(global::System.Runtime.InteropServices.HandleRef jarg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_PixelData_GenerateUrl")]
+            public static extern string GenerateUrl(IntPtr handle);
         }
     }
 }
```

Comment:
from next time, please add new line here~
(https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)

Reply:
!!! Sorry I couldn't checked it.